template.tpl

```diff
@@ -619,6 +763,167 @@ ___TEMPLATE_PARAMETERS___
         "checkboxText": "EU Data Residency",
         "type": "CHECKBOX"
       },
+      {
+        "help": "Check this box to enable default event tracking. \u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sdks/browser-2/#tracking-default-events\"\u003eRead more\u003c/a\u003e.",
+        "defaultValue": true,
+        "simpleValueType": true,
+        "name": "defaultEventTracking",
+        "checkboxText": "Enable default event tracking",
+        "type": "CHECKBOX",
+        "subParams": [
+          {
+            "type": "GROUP",
+            "name": "defaultEventTrackingOptions",
+            "displayName": "Default Event Tracking Options",
+            "groupStyle": "ZIPPY_CLOSED",
+            "subParams": [
+              {
+                "type": "CHECKBOX",
+                "name": "detAttribution",
+                "checkboxText": "Enable attribution tracking",
+                "simpleValueType": true,
+                "subParams": [
+                  {
+                    "type": "GROUP",
+                    "name": "attributionOptions",
+                    "groupStyle": "NO_ZIPPY",
+                    "subParams": [
+                      {
+                        "type": "TEXT",
+                        "name": "attributionInitialEmptyValue",
+                        "displayName": "Initial empty value",
+                        "simpleValueType": true,
+                        "defaultValue": "EMPTY",
```

Comment:
Same for this default value, i think we should leave the form field empty and handle behind the scenes